virtwho/config.py

```diff
@@ -873,9 +906,30 @@ def __setitem__(self, key, value):
     def _pre_validate(self):
         # Used to allow virttype to be provided for type (if type was not given)
         if 'virttype' in self._values and 'type' not in self._values:
-            self._values['type'] = self._values['virttype']
+            self['type'] = self._values['virttype']
```

Comment:
It seem you try to do some refactoring 👍 (renaming variables), but I'm I'm little bit confused. Should it be `virttype` or `virtType` or `virt_type`? I can still find three names of key/variable for probably the same thing. Please finish your refactoring effort :-).

Reply:
Will do!